synnax/pkg/version/GetVersion.go

```diff
@@ -0,0 +1,16 @@
+package version
+
+import (
+	"embed"
+	"fmt"
+	"io/ioutil"
+)
+
+//go:embed VERSION
+var fs embed.FS
+
+func GetVersion() string{
```

Comment:
When you import this, you'd call `version.GetVersion()`, which is redundant. Instead, rename to `version.Get()` or (and this is technically considered OK by the go style standards) `version.Version()`